jdbc-v2/src/main/java/com/clickhouse/jdbc/internal/DriverProperties.java

```diff
@@ -22,8 +22,8 @@ public enum DriverProperties {
      * query settings to be passed along with query operation.
      * {@see com.clickhouse.client.api.query.QuerySettings}
      */
-    DEFAULT_QUERY_SETTINGS("default_query_settings", null);
-
+    DEFAULT_QUERY_SETTINGS("default_query_settings", null),
+    BEARER_TOKEN("jdbc_bearer_token", null);

```

Comment:
if there is a property in Client - we do not need this.
user will input "bearer_token" and it will be passed to the client.